src/libraries/Microsoft.PowerFx.Core/Public/Types/FormulaType.cs

```diff
@@ -87,7 +86,6 @@ internal FormulaType(DType type)
             { new DName("Number"), Number },
             { new DName("Decimal"), Decimal },
             { new DName("Text"), String },
-            { new DName("Hyperlink"), Hyperlink },
```

Comment:
>{ new DName("Hyperlink"), Hyperlink }, [](http://example.com/codeflow?start=12&length=38)

Please check with @jas-valgotar  if this is okay to do as this is also used in [https://github.com/microsoft/Power-Fx/blob/53da8f65899eaf8290eb09d6e9ca16f1efeedd56/src/libraries/Microsoft.PowerFx.Json/Public/Types/UserProvidedTypeDefinitions/FormulaTypeSchema.cs#L91](https://github.com/microsoft/Power-Fx/blob/53da8f65899eaf8290eb09d6e9ca16f1efeedd56/src/libraries/Microsoft.PowerFx.Json/Public/Types/UserProvidedTypeDefinitions/FormulaTypeSchema.cs#L91)